free-1.md

```diff
@@ -57,12 +70,16 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+It's difficult to tell at this point, but my guess would be around 50% language design.  In terms of language design, the creator would have to design a way to accept an ASCII representation of a drum fill.  The creator would also have to create user-friendly syntax for commands that would alter the drum fill.   In terms of "systems", algorithms would be needed to develop to properly randomize elements of a drum fill based on given parameters.  The creator would also have to come up with a robust representation of a drum fill internally (a potential system for this was explained in the "Interface" section), and a playback system would need to be created, likely with MIDI.
 
-### Scope
+###  Scope
 _How big an idea is this? How ambitious is this project?_
 
+This DSL idea is relatively ambitious, as it would require a few separate processes to be implemented, none of which are trivial.  These processes include drum fill "parsing" (reading in fills that users create), algorithms to alter these drum fills, and properly handling MIDI.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This might be a good idea because it's very extensible.  The creator could start off with support for just one "instrument" (e.g. a single track for just a hi-hat) and a basic "randomize" algorithm.  This would allow the creator to flesh out the "language" aspects of the DSL while having a fully-functioning vertical slice of what the final DSL might look like.  From there, for the purposes of the project it shouldn't matter too much how far along the creator is able to get in adding multiple instruments or more complex "randomize" algorithms.
```

Comment:
I agree, it seems like you could live with only a few parts of the problem done as an MVP, and then extend it by adding more features if you had time.